arviz/data/io_cmdstanpy.py

```diff
@@ -50,6 +50,9 @@ def __init__(
 
         self.save_warmup = rcParams["data.save_warmup"] if save_warmup is None else save_warmup
 
+        if self.log_likelihood is None and "log_lik" in self.posterior.stan_vars_cols:
```

Comment:
I'm not against this, but is this something we want users to do?

Reply:
I think so, it seems to be a convention already in use on the R side, see http://mc-stan.org/loo/reference/extract_log_lik.html for example.